### ISSUE TITLE
Major version updates don't work

### ISSUE THREAD

**Issue by mikeburgh** [Pilaton/vscode-npm-lens-support#1]
If the update is a minor (eg 2.0.0 to 2.1.1) it works, but a major update like 2.0.0 to 3.0.0 the update command does nothing, it should run npm install instead.

**Comment by Pilaton** [comment]
Resolved. Thanks for the report!

**Comment by Pilaton** [comment]
Resolved. Thanks for the report!

**Comment by Pilaton** [comment]
Resolved. Thanks for the report!

**Comment by mikeburgh** [comment]
Installed 3.0.4, but still not working for major updates..

It just runs npm update @typescript-eslint/parser which does not push it to the latest major version which is 7.1.0
The package.json has it listed as: 
"@typescript-eslint/parser": "^5.31.0",

I think it might need to uninstall first, then install ? 

Also it would be nice to include the unpacked size in the table summary, so you can see big packages easily rather than needing to click into each one.

**Comment by Pilaton** [comment]
@mikeburgh, I just checked the update from "@typescript-eslint/parser": "^5.31.0", to the latest version - everything works fine. 
VSCode should have automatically updated the extension to the latest version.  

Is the problem occurring with the "npm" package manager, or some other package manager?

**Comment by mikeburgh** [comment]
Sorry something was up with vscode, it was showing 3.0.4 but still using update.. restart seems to have cleared it and it's now correctly using npm install.

A few other things I noticed though: 
- If you update in devDependencies it reloads and takes you back to Dependancies tab, would be nice to stay where you are (eg working your way down through a list)
- Re use terminals, rather than create a new one for each command.

Otherwise, great extension, best of the ones that do this that I have found!
